Dockerfile

```diff
@@ -14,16 +14,13 @@ ENV PORT 8080
 ENV BASE_URL ""
 ENV SWAGGER_JSON_URL ""
 
-COPY ./docker/nginx.conf ./docker/cors.conf /etc/nginx/
+COPY --chown=nginx:nginx --chmod=0666 ./docker/nginx.conf ./docker/cors.conf /etc/nginx/
```

Comment:
Is this `--chown` still required?

Reply:
As long as user/group/other have the same rights it's not needed. With or without the --chown this'll work the same.